README.md

```diff
@@ -93,7 +94,8 @@
 - [Solidity by Example](https://solidity-by-example.org) - Basic examples on Solidity.
 - [Web3 by Example](https://www.web3byexample.com/) - Basic examples on Ethers with JavaScript
 - [Vyper by Example](https://vyper-by-example.org/) - Basic examples on Vyper.
-- [Yul By Example](https://github.com/Perelyn-sama/yul_by_example) - Basic examples on Yul. 
+- [Merkle Proof Example](https://github.com/8gen/erc721a-merklerproof-template) — Usage example for Merkle Proof on Solidity and Javascript
```

Comment:
This repo deserves a Readme that explains how to use the template or what it is being an example of.

So, we have 2 choices here. First, you can add a Readme that describes how to use that example and we merge this PR. Second, we remove the Merkle proof example from this PR and merge and you can create a new PR for the Merkle proof example which can be merged after adding a Readme.

Reply:
I prefer to add README, I thought about it. That would be really more useful. You are right.